shell/platform/fuchsia/flutter/accessibility_bridge_unittest.cc

```diff
@@ -806,8 +806,9 @@ TEST_F(AccessibilityBridgeTest, BatchesLargeMessages) {
   RunLoopUntilIdle();
 
   EXPECT_EQ(0, semantics_manager_.DeleteCount());
+
   EXPECT_TRUE(6 <= semantics_manager_.UpdateCount() &&
-              semantics_manager_.UpdateCount() <= 10);
```

Comment:
Have we asked anyone on the a11y team why this might be the case?

This is not expected behavior to change, and it would be good to know if we missed a breakage here.

We should not be silently fixing these things without talking to the appropriate platform team.  This is how breakages go untriaged and undiagnosed....

Reply:
Ahh, I looked into it more and this is probably because of https://source.corp.google.com/github/flutter/engine/shell/platform/fuchsia/flutter/accessibility_bridge.cc;l=589?q=accessibility_bridge.cc&sq=package:github

So, I guess updating to 12 is fine for now.

These checks on UpdateCount() are going to make the test flaky though, as you've seen here.  Filed https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=87770 to address this in a holistic way.

Reply:
I did. They said that the batch size will change when they add more fields to the semantics update field since the fidl message size has a limit. I agree that this is not ideal but do you think we should block the SDK roll based on it?